packages/dashmate/configs/migrations.js

```diff
@@ -419,16 +419,42 @@ module.exports = {
     return configFile;
   },
   '0.24.0-dev.13': (configFile) => {
+    configFile.configs.base.platform.drive.tenderdash.genesis = systemConfigs.base.platform
```

Comment:
Shouldn't it be inside 0.24.0-dev.14 instead?

Reply:
Yeah, you probably right